libcxx/test/std/containers/sequences/vector.bool/exceptions.pass.cpp

```diff
@@ -0,0 +1,130 @@
+//===----------------------------------------------------------------------===//
```

Comment:
Let's rename this file to `ctor.exceptions.pass.cpp` since we're only testing constructors.

Actually, writing this comment made me realize that we already have `libcxx/test/std/containers/sequences/vector.bool/ctor_exceptions.pass.cpp`. What's up with that test? We should merge this new test into that existing test instead. Do you have thoughts about that?

Reply:
Oops, I should have noticed this file earlier—it would have simplified my PR.
 I just quickly reviewed this file `libcxx/test/std/containers/sequences/vector.bool/ctor_exceptions.pass.cpp` and found  it has similarly flaw as in the exception test for `std::vector`: the same `Allocator` class is used in `ctor_exceptions.pass.cpp` does not trigger exception at all, making some tests checking for nothing for `std::vector<bool>`. Also, this test file also provided duplicate definitions for the `Allocator` and `Iterator` classes. I believe we still need to refactor these common classes into a file such as `libcxx/test/std/containers/sequences/vector/common.h`, as you suggested earlier. I will work on this first.